input/witnessgen.go

```diff
@@ -32,7 +33,7 @@ type WitnessType interface {
 	// WitnessType if it would be included in a tx. It also returns if the
 	// output itself is a nested p2sh output, if so then we need to take
 	// into account the extra sigScript data size.
-	SizeUpperBound() (int, bool, error)
+	SizeUpperBound() (lntypes.WeightUnit, bool, error)
```

Comment:
Do we want to rename this function to `WeightUpperBound`? I don't mind it now that we have the explicit return type but could be a nice way to keep things consistent.